plugins/catalog-react/src/components/EntityPeekAheadPopover/EntityPeekAheadPopover.tsx

```diff
@@ -146,61 +137,58 @@ export const EntityPeekAheadPopover = (props: EntityPeekAheadPopoverProps) => {
           }}
           onMouseLeave={handleOnMouseLeave}
         >
-          <>
-            {error && <ResponseErrorPanel error={error} />}
-            <Card>
+          <Card>
+            <CardContent>
+              {error && <ResponseErrorPanel error={error} />}
```

Comment:
moved into the content because otherwise the paddings become broken

Reply:
Its hard to get this type of thing right. Well done!